ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml

```diff
@@ -21,3 +21,52 @@ LCM_SS:
   implements:
   - LCM
   - SS
+
+LCM_OS:
+  description: "Standard lighting for depicting the zone occupancy."
+  is_canonical: true
+  implements:
+  - LCM
+  - OS
+  
+LCM_IM:
+  description: "Standard lighting with illuminance sensor."
+  is_canonical: true
+  implements:
+  - LCM
+  - IM
+
+LCM_LSC:
+  description: "Standard lighting with switch control."
+  is_canonical: true
+  implements:
+  - LCM
+  - LSC
+
+LCM_PWM:
```

Comment:
do we not have any other data for the device? if this is a control module shouldnt there be control?

Reply:
The single canonical has been deleted and the unified canonical LGRP_OS_IM_LSC_LC_PWM_SCZ has been created.